doc/ref_model/chapters/chapter07.md

```diff
@@ -630,8 +627,9 @@ The Platform is assumed to provide configurable alerting and notification capabi
 |---|----|----|
 | req.sec.oss.001 | Open source code **must** be inspected by tools with various capabilities for static and dynamic code analysis. |  |
 | req.sec.oss.002 | The CVE(Common Vulnerabilities and Exposures) **must** be used to identify vulnerabilities and their severity rating for open source code part of Cloud Infrastructure and workloads software.  | https://cve.mitre.org/ |
-| req.sec.oss.003 | A dedicated internal isolated repository separated from the production environment **must** be used to store vetted open source content. |  |
-| req.sec.oss.004 | A Software Bill of Materials (SBOM) **should** be provided or build, and maintained to identify the software components and their origins. | Inventory of software components, https://www.ntia.gov/SBOM. | 
+| req.sec.oss.003 | High severity rated vulnerabilities must be fixed. Refer to the CVSS (Common Vulnerability Scoring System) to know a vulnerability score.  | https://www.first.org/cvss/ |
```

Comment:
Maybe rather 'find a vulnerability score'?

Reply:
@ildikov Rather than 'find', I prefer 'evaluate' or 'calculate' as there is a part of the score which is specific to the use of the product (or software) affected by the vulnerability. Does it work for you?

Reply:
@karinesevilla: "| req.sec.oss.003 | High severity rated vulnerabilities must be fixed. Refer to the CVSS (Common Vulnerability Scoring System) to know a vulnerability score.  | https://www.first.org/cvss/ |"
is unclear and does not sound good. Are you trying to say that in order to fix vulnerabilities one needs to refer to CVSS?  If yes, then the sentence needs to be more clear.  If not, what are we trying to say in the second sentence?

Reply:
I think the sentence needs to be clarified as Walter indicated in his comment as well. It should be a minor change I believe in terms of someone needs to look at CVSS in order to be able to determine the severity of their vulnerabilities and act accordingly. I'm not deeply familiar with the actual process, so I may not proposed the best word, apologies for that.

Reply:
@ildikov @walterkozlowski I have added more information about CVE and CVSS in 'Vulnerability Management', especially about CVSS scores and rates. It should clarify how it works and answer your questions. I have also slighlty modified the requirement, but I do not think it is necessary to add more text there.